### PR TITLE
Remove color ops from v8

### PR DESCRIPTION
**Pull request by lucaswoj**
cc @jfirebaugh 

ref #308

### CODE REVIEW

**Comment by lucaswoj** [comment]
Sorry about the chaos between #330 and this PR. Everything's in ship shape and ready for :eyes: now.

File: lib/validate/parsed.js [hunk 1]
```diff
@@ -345,35 +345,19 @@ module.exports = function(style, reference) {
...
-            if (parseCSSColor(val) === null) {
-                error(key, val, 'color expected, "%s" found', val);
-            }
-        } else {
-            validate.enum(key + '[0]', val[0], reference.color_operation);
-
-            if (isNaN(val[1])) {
-                error(key, val, 'number expected, "%s" found', val[1]);
-            }
+        if (reference.$version <= 7 && val[0] === '@') {
```
Comment (jfirebaugh) [inline lib/validate/parsed.js]: I believe constant values will have already been resolved by this point.

File: migrations/v8.js [hunk 2]
```diff
@@ -1,5 +1,22 @@
 'use strict';
 
+var Reference = require('../reference/v8');
+var ColorOps = require('color-ops');
```
Comment (jfirebaugh) [inline migrations/v8.js]: No need to support color-ops in the migration. These weren't available in v7 and there are no v8 styles in the wild that would need to be migrated.

Comment (lucaswoj) [inline migrations/v8.js]: :+1:

File: lib/validate/parsed.js (hunk 1 above)
Comment (lucaswoj) [inline lib/validate/parsed.js]: :+1:

**Pull request merged**